backend/cpp/llama/grpc-server.cpp

```diff
@@ -495,6 +495,16 @@ struct llama_server_context
         }
     }
 
+    llama_client_slot* get_active_slot() {
+        for (llama_client_slot& slot : slots) {
+            // Check if the slot is currently processing
+            if (slot.is_processing()) {
```

Comment:
if we replace that with slot.available() - if there was at least one inference we get statistics for the last call

if not - we get a 'Server error error="json: unsupported value: NaN"'